guides/administration/using-nsenter-to-access-a-node-without-ssh.md

```diff
@@ -0,0 +1,41 @@
+# Using nsenter to access a node without SSH
+
+In addition SSH, [nsenter](https://github.com/jpetazzo/nsenter) can be used to gain root access to a node. This method does NOT require any password or authentication, so be sure that your Kontena grid users are trusted.
+
+First you need to install the `nsenter` as a daemon service so that it runs on each node. There is a sample stack in the Kontena Stack Registry:
+
+    $ kontena stack registry pull matti/nsenter
+      stack: matti/nsenter
+      version: 0.0.1
+      description: kontena node nsenter
+
+      services:
+        nsenter:
+          image: walkerlee/nsenter
+          deploy:
+            strategy: daemon
+          command: "-t 1 -m -u -i -n sleep 2073600"
```

Comment:
Maybe add couple of comments what that command actually does. I'd guess 97% of readers will not get it without any explanation.

Reply:
I would, but I really don't know what it does, I've just been using it. The only clarification I could write is that "based on it's name it does something with docker namespaces to gain access to the container" - if you understand more, please add a better clarification. I'm not interested to dig deeper into the inner workings, I'm just using nsenter almost daily because it's magic and convenient.

I also think that guides can be shallow just to get 'em done fastly and to inspire people. 99.9999999% of the readers don't know that such a trick exists without this guide.

Reply:
It attaches itself to host namespaces and thus allows you to operate on the host space.
(Simplified as hell, but makes the point I guess :) )